CONTRIBUTING.md

```diff
@@ -125,12 +145,14 @@ server {
 }
 ```
 
-And add a record to `/etc/hosts` to map the `docs.apify.loc` hostname to localhost:
+Add a record to `/etc/hosts`, which maps the `docs.apify.loc` host to a localhost:
```

Comment:
nit: `hostname` was probably more correct, from what I read, `host` is also supposed to contain the port, while `hostname` is just the name. but maybe not, haven't been googling that hard (and was happy with the first result that confirmed what I thought)